sdk/python/feast/infra/gcp.py

```diff
@@ -93,7 +93,7 @@ def online_write_batch(
     ) -> None:
         client = self._initialize_client()
 
-        pool = ThreadPool(processes=10)
+        pool = ThreadPool(processes=40)
```

Comment:
Do we need 40 here? Where does the ROI drop off?

Reply:
I saw an improvement from 20 -> 40, but didn't see any improvement from 40 -> 50.